ThirdPartyAdapters/vungle/build.gradle

```diff
@@ -15,6 +15,8 @@ buildscript {
 
 allprojects {
     repositories {
+        maven { url 'https://plugins.gradle.org/m2/' }
```

Comment:
can we clean this, think this was initially added for beta dependency

Reply:
Done.